src/frontend/screens/ObservationDetails/Question.tsx

```diff
@@ -8,19 +7,17 @@ import TextArea from "./TextArea";
 import type { Field } from "../../context/ConfigContext";
 
 export type QuestionProps = {
-  field: Field,
-  value: any,
-  onChange: (value: any) => any,
+  field: Field;
+  value: any;
+  onChange: (value: any) => any;
```

Comment:
```suggestion
  onChange: (value: any) => void;
```

Reply:
ideally would fix up this `QuestionProps` type to be more specific about what `value` is (e.g. `string`, `number`, etc) but more of a nit than anything